boggle.js

```diff
@@ -1,26 +1,45 @@
 function newBoggleBoard() {
-  return [
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-    ['_', '_', '_', '_'],
-  ];
+
+  let boggleBoard = [];
+  for(i=0;i<4;i++)
+  {
+    boggleBoard.push(['_', '_', '_', '_'])
+  }
+  return boggleBoard;
 }
 
+
 /**
  * Prints out a boggle board.
  */
+
+let dice = ['AAEEGN','ELRTTY','AOOTTW','ABBJOO','EHRTVW','CIMOTU','DISTTY','EIOSST','DELRVY','ACHOPS','HIMNQU','EEINSU','EEGHNW','AFFKPS','HLNNRZ','DEILRX'];
+
 function printBoggleBoard(boggleBoard) {
   for (let row of boggleBoard) {
     console.log(row.join(' '));
   }
 }
 
+function randomInteger(min, max) {
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+
 /**
  * Shake a boggle board and fill it with letters.
  */
 function shake(boggleBoard) {
-  // This is your job. :)
+  for (let row of boggleBoard)
+  {
+    for (i=0;i<4;i++)
+    {
+      let index = dice[(randomInteger(0, dice.length - 1))];
+      row[i] = index[Math.floor(Math.random() * (6))];
```

Comment:
1. Use the `randomInteger` function you wrote.
1. You don't need the parens around `6`.
1. Why hard code `6` here but not in the line above?